zerver/webhooks/gocd/view.py

```diff
@@ -9,13 +9,26 @@
 from zerver.lib.typed_endpoint import JsonBodyPayload, typed_endpoint
 from zerver.lib.validator import WildValue, check_string
 from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.lib.webhooks.git import get_short_sha
 from zerver.models import UserProfile
 
-MESSAGE_TEMPLATE = """\
-Author: {}
-Build status: {} {}
-Details: [build log]({})
-Comment: {}"""
+COMMIT_INFO_TEMPLATE = (
+    """Triggered on [`{commit_details}`]({commit_link}) on branch `{branch_name}`."""
+)
+TOPIC_TEMPLATE = "{pipeline} / {stage}"
+
+SCHEDULED_BODY_TEMPLATE = """
+**Pipeline** {pipeline}/{stage}: {status}.
+- **Commit**: {commit_details}
+- **Started**: {start_time}
+"""
+
+COMPLETED_BODY_TEMPLATE = """
+**Build** {pipeline}/{stage}: {status} {emoji}.
```

Comment:
I'd lead with the emoji, put the status in the bold section ...

`{emoji} **Build {status}**: {pipeline} / {stage}`